packages/rpc-core/src/rpc-methods/getBlockProduction.ts

```diff
@@ -0,0 +1,30 @@
+import { Base58EncodedAddress } from '@solana/keys';
+import { Commitment, RpcResponse, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+type NumberOfLeaderSlots = number;
+type NumberOfBlocksProduced = number;
```

Comment:
(Y). Looked at source code. Saw these were both `usize`.

Reply:
Uh, oh damn. Yes on 64 bit machines it could be. I got this wrong elsewhere; I’m going to have to go fix that now!